crates/nu-std/std/mod.nu

```diff
@@ -29,11 +29,6 @@ export module ./dirs {
 
 # Workaround for #13403 to load export-env blocks from submodules
 export-env {
-    # log
-    $env.NU_LOG_FORMAT = $env.NU_LOG_FORMAT? | default "%ANSI_START%%DATE%|%LEVEL%|%MSG%%ANSI_STOP%"
-    $env.NU_LOG_DATE_FORMAT = $env.NU_LOG_DATE_FORMAT? | default "%Y-%m-%dT%H:%M:%S%.3f"
-    
-    # dirs
-    $env.DIRS_POSITION = 0
-    $env.DIRS_LIST = [($env.PWD | path expand)]
+    use std/log []
```

Comment:
what's the point of `use std/log []` vs `use std/log *` or `use std/log`?

Reply:
Honestly, I just reverted it to its original form.  At some point I had mucked with it and broke it.

I think the point is to eval the `export-env` block without actually importing anything (an empty list of command imports).

Interestingly, this PR reduced load times of `use std *` by another 2ms. I really don't see anything in my changes that would have made it faster, but I'm not complaining.